### PR TITLE
Fix vpiAlways handling

### PR DESCRIPTION
**Pull request by RRozak**

### CODE REVIEW

File: src/UhdmAst.cpp [hunk 1]
```diff
@@ -2122,8 +2122,7 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
     }
     case vpiAlways: {
```
Comment (rkapuscik) [inline src/UhdmAst.cpp:2170]: Move this to a `process_*` function.

File: src/UhdmAst.cpp [hunk 2]
```diff
@@ -2149,18 +2148,9 @@ AstNode* visit_object(vpiHandle obj_h, UhdmShared& shared) {
...
-            visit_one_to_one({vpiStmt}, obj_h, shared, [&](AstNode* node) { body = node; });
-        }
-
-        return new AstAlways(make_fileline(obj_h), alwaysType, senTree, body);
+        // Body of always statement
+        visit_one_to_one({vpiStmt}, obj_h, shared, [&](AstNode* nodep) { bodyp = nodep; });
+        return new AstAlways(make_fileline(obj_h), alwaysType, nullptr, bodyp);
     }
     case vpiEventControl: {
         AstSenItem* senItemRoot = nullptr;
```
Comment (rkapuscik) [inline src/UhdmAst.cpp]: Seems like a good opportunity to:
1. Fix the naming convention.
2. Refactor this to a `process_*` function.

**Review by rkapuscik — CHANGES_REQUESTED** [review]
Looks OK, let's wait for the CI. Also, seems like a good opportunity to clean up the old code a bit.

File: src/UhdmAst.cpp (hunk 2 above)
Comment (RRozak) [reply src/UhdmAst.cpp]: Done

File: src/UhdmAst.cpp (hunk 1 above)
Comment (RRozak) [reply src/UhdmAst.cpp:2170]: Done

**Review by rkapuscik — APPROVED** [review]

**Pull request merged**